pygfx/renderers/wgpu/wgsl/grid.wgsl

```diff
@@ -64,9 +66,21 @@ fn vs_main(in: VertexInput) -> Varyings {
         //  | / /           \\|
         //  x-----------------x
 
+        // Get scale multiplier for near quad.
+        // With an orthographic camera, we use the scale factors for x and y (camera width and height).
+        // For perspective cameras we use the distance to the plane and assume an aspect ratio of 1.
+        var near_multiplier = vec2<f32>(1.0);
+        if is_orthographic() {
```

Comment:
to improve understanding, maybe this if statement should be around the whole computation for `distance_cam_to_grid`.

I understand that it won't help performance, but it may help the understanding in saying that:

"Our computation for the distance between the camera to the grid is only valid when the camera isn't an orthographic projection"

My understanding is that the nature of the orthographic projection makes the camera matrix illposed in certain situations.